src/pages/PositionDetails/Contents.tsx

```diff
@@ -70,34 +78,45 @@ export const Contents = ({ position }: Props) => {
       .map((value, index) => (value === '1' ? index + 1 : 0))
       .filter((n) => !!n)
   })
-  const dropdownItems = [
-    {
-      onClick: () => {
-        logger.log('Redeem')
+  const dropdownItems = useMemo(() => {
+    return [
+      {
+        onClick: () => {
+          setValue(id)
+          history.push(`/redeem`)
+        },
```

Comment:
Lets move this function to an `useCallback` to prevent creating a new function every time the component is rendered